stable/mysql/templates/configurationFiles-configmap.yaml

```diff
@@ -3,6 +3,7 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ template "mysql.fullname" . }}-configuration
+  namespace: {{ .Release.Namespace }}
```

Comment:
doesnt helm do this already?

Reply:
only at deployment time, so if you are using purely as a template tool (for instance in Spinnaker) it doesn't do that.